index.html

```diff
@@ -35,10 +50,10 @@ <h1 class="chloe-smart">Chloe Smart</h1>
 		
 		<div class="text">
 			
-			<h2 class="about-me">About Me</h2>
+			<h2 class="about-me-title">About Me</h2>
 		
 			<p class="about-me">Hello! I'm Chloe, I am an aspiring Software Developer. Previously I have worked as a Printed Textile Designer, and for the past few years I have owned my own illustration & design business.</p>
-				<p class="about-me">I first got into coding when building & designing my own website for my business, I loved thinking about how to create the best experience for the user & problem solving how to make my website work well</p>
+				<p class="about-me">I first got into coding when building & designing my own website for my business, I loved thinking about how to create the best experience for the user & problem solving how to make my website work to a high quality</p>
```

Comment:
below, the image 'me' could perhaps have a more descriptive alt for accessabliilty?